src/consts/values.ts

```diff
@@ -7,3 +7,5 @@ export const COSMOS_ZERO_ADDRESS = 'cosmos10000000000000000000000000000000000000
 export const SOL_SPL_NOOP_ADDRESS = 'noopb9bkMVfRPU8AsbpTUg8AQkHtKwMYZiFUjNRtMmV';
 export const SOL_IGP_QUOTE = '10000';
 export const COSM_IGP_QUOTE = '100';
+// Placeholder chain for use with cosmos-kit wallet hooks that require a chain name
+export const PLACEHOLDER_COSMOS_CHAIN = 'cosmoshub';
```

Comment:
maybe celestia for usenexus?

Reply:
It works better to be a chain wallets already have. I don't think this does what you'd expect. It's really just a hack to work around limitations of cosmos-kit being used in a 'sometimes not cosmos' dapp.